examples/node/HistoricalDataRequest.js

```diff
@@ -2,7 +2,7 @@
 var https = require('https');
 var fs = require('fs');
 
-var host = process.argv[2] || '127.0.0.1';
+var host = "http-api.openbloomberg.com";
```

Comment:
Can you update this to be:
```javascript
var host = process.argv[2] || "http-api.openbloomberg.com";
```
please?

It would keep with the original intent of the example with just updating the default host.  Thanks!